BUILD

```diff
@@ -1445,3 +1446,139 @@ internal_gen_kt_protos(
     visibility = ["//java:__subpackages__"],
     deps = [":kt_proto3_unittest"],
 )
+
+################################################################################
+# Packaging rules
+################################################################################
+
+# Files included in all source distributions
+pkg_files(
+    name = "common_dist_files",
+    srcs = glob([
+        "*.bzl",
+        "cmake/*.cmake",
+        "cmake/*.in",
+        "editors/*",
+
+        # Several of these files are generated by autogen.sh, so using
+        # glob() lets us ignore them if they are missing. (This is not good
+        # practice, though.)
+        "Makefile.*",  # Makefile.am is always present, though
```

Comment:
Since Makefile.am is always present, can we put it in the exact matches below?

Reply:
Done.